jstreegrid.js

```diff
@@ -216,7 +218,7 @@
 					gs.gridcontextmenu = false;
 				}
 				// find which column our tree shuld go in
-				for (i=0;i<s.columns.length;i++) {
+        for (var i = 0, len = s.columns.length;i<len;i++) {
```

Comment:
You _really_ like setting a `var len` to the length in loops, don't you? I don't care, but I don't think it buys all that much. Leave it for now, I guess.